src/BikramSambat.ts

```diff
@@ -22,4 +22,13 @@ export default class BikramSambat {
     this.month = month
     this.day = day
   }
+
+  public getPreviousYear(): number {
+    return this.year - 1
+  }
```

Comment:
just in case this.year is undefined (could be in case of invalid date) we should return `NaN` instead
```suggestion
  public getPreviousYear(): number {
    return this.year ? this.year - 1 : NaN
  }
```